src/components/ui/password.tsx

```diff
@@ -0,0 +1,25 @@
+"use client";
+
+import { forwardRef, useState } from "react";
+
+import { EyeClosedIcon, EyeOpenIcon } from "@radix-ui/react-icons";
+
+import { Input, InputProps } from "@/components/ui/input";
+
+const Password = forwardRef<HTMLInputElement, InputProps>((props: InputProps, ref) => {
+  const [visible, setVisible] = useState(false);
+
+  const toggleVisible = () => setVisible(!visible);
+
+  return (
+    <div className="relative">
+      <Input {...props} ref={ref} type={visible ? "text" : "password"} />
+      <div className="absolute bottom-2 right-5" onClick={toggleVisible}>
+        {visible ? <EyeOpenIcon /> : <EyeClosedIcon />}
+      </div>
```

Comment:
Hi, @Badra00, could you please add tab navigation? While entering a password, could you also include tabIndex and an onKeyDown event to toggle the password view for improved accessibility?